www/tasks/assets.js

```diff
@@ -27,14 +30,19 @@ gulp.task('assets', function() {
 	dirs.forEach(function(dir) {
 		var cwd = path.join(srcAssetsPath, dir);
 		var dest = path.join(destAssetsPath, dir);
-		var useSymlink = true; // @TODO: When it's release copy otherwise symlink.
```

Comment:
Oh woops! Totally didn't mean to remove that, I had to toggle the Boolean temporarily but must have removed by mistake 🙈